packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -0,0 +1,343 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  color: $text-body;
+  font-family: $body-font-family;
+  font-size: ($base-font-size / 16px) * 100%;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: $base-line-height;
+  font-feature-settings: "lnum", "pnum";
+  text-decoration-skip-ink: all;
+
+  /* stylelint-disable selector-max-type, selector-max-class, selector-class-pattern */
+  abbr,
+  &.abbr {
+    border-block-end: 1px dashed $color-primary-dark;
+    text-decoration: none;
+  }
+
+  em,
+  &.em {
+    font-style: italic;
+
+    > em {
+      text-decoration: underline;
+    }
+  }
+
+  strong,
+  &.strong {
+    font-weight: 600;
+
+    > strong,
+    > .strong {
+      text-decoration: underline;
+    }
+  }
+
+  sub,
+  &.sup {
+    font-size: ms(-2);
+    line-height: 1;
+    vertical-align: super;
+  }
+
+  sub,
+  &.sub {
+    font-size: ms(-2);
+    line-height: 1;
+    vertical-align: sub;
+  }
+
+  blockquote,
+  &.blockquote {
+    max-width: $max-line-length;
+    margin-block-start: 0;
+    margin-block-end: $type-margin;
+    margin-inline: 0;
+    padding-block: 0;
+    padding-inline-end: 0;
+    padding-inline-start: $spacing-s;
+    border-inline-start: 3px solid $border-color-display;
+
+    &:last-child {
+      margin-block-end: 0;
+    }
+  }
+
+  p,
+  &.p {
+    max-width: $max-line-length;
+    margin-block-start: 0;
+    margin-block-end: $type-margin;
+    margin-inline: 0;
+
+    details &,
+    .details & {
+      font-size: ms(0);
+    }
+
+    &:last-child {
+      margin-block-end: 0;
+    }
+  }
+
+  pre,
+  &.pre {
+    margin-block-start: 0;
+    margin-block-end: $type-margin;
+    margin-inline: 0;
+    font-family: $mono-font-family;
+    white-space: pre-wrap;
+    tab-size: 2;
+
+    &:last-child {
+      margin-block-end: 0;
+    }
+  }
+
+  cite,
+  &.cite {
+    font-style: italic;
+  }
+
+  del,
+  &.del {
+    display: inline-block;
+    background: $color-danger-light;
+  }
+
+  dfn,
+  &.dfn {
+    font-style: italic;
+  }
+
+  ins,
+  &.ins {
+    display: inline-block;
+    background: cv("green", "000");
+  }
+
+  ins::before,
+  ins::after,
+  &.ins::before,
+  &.ins::after {
+    @include is-visually-hidden;
+  }
+
+  ins::before,
+  &.ins::before {
+    content: attr(data-a11y-start);
```

Comment:
why do we need this? Based on what I've read this approach will now be replaced by having consumers compose our `<ScreenReaderText />` components yes?